### DISCUSSION TITLE
Plugins to be loaded in headless/server mode

### DISCUSSION THREAD

**Discussion by hqnghi88** [gama-platform/gama#68] [General]
I wonder if this activation is mandatory for headless mode?
> JAI : ImageIO extensions   
Currently, i disable the java2d, opengl, ui feature, models library in headless mode, it seems to have no issue.
@AlexisDrogoul

**Comment by AlexisDrogoul** [comment]
JAI extensions are used by GeoTools and seem necessary for reading some raster files (like tiff files).

**Comment by RoiArthurB** [comment]
@hqnghi88 In complement to Alexis' message, it is needed by the headless to take screenshots of the running simulation :)

**Comment by hqnghi88** [comment]
Thankyou, good to know.
